eslint.config.js

```diff
@@ -5,9 +5,10 @@ import eslintPluginImport from 'eslint-plugin-import'
 import eslintPluginReact from 'eslint-plugin-react'
 // @ts-expect-error no types are available for this plugin
 import eslintPluginReactHooks from 'eslint-plugin-react-hooks'
+import eslintPluginVitest from '@vitest/eslint-plugin'
 
-const REACT_FILES_MATCH =
-  'packages/{tuono,tuono-router}/**/*.{js,mjs,cjs,jsx,mjsx,ts,tsx,mtsx}'
+const JS_EXTENSIONS_MATCH = 'js,mjs,cjs,jsx,mjsx,ts,tsx,mtsx'
```

Comment:
```suggestion
const JS_EXTENSIONS_MATCH = ['js','mjs','cjs','jsx','mjsx','ts','tsx','mtsx'].join(',')
```

Reply:
I don't see any particular advantage of this, but I don't have a strong opinion so I will apply your version.